x-pack/plugins/security_solution_serverless/server/task_manager/usage_reporting_task.ts

```diff
@@ -126,15 +120,21 @@ export class SecurityUsageReportingTask {
     const [{ elasticsearch }] = await core.getStartServices();
     const esClient = elasticsearch.client.asInternalUser;
 
-    const lastSuccessfulReport =
-      taskInstance.state.lastSuccessfulReport && new Date(taskInstance.state.lastSuccessfulReport);
+    const epochDate = new Date();
+    epochDate.setFullYear(1969);
```

Comment:
Q: Why 1969? If there's a specific reason perhaps a good idea to mention it in a comment for posterity.

Reply:
I just need unix epoch and this was easier than setting day/month/year to unix epoch. Good point on the comment though. I plan to add some more tests so will add the comments in that PR.